public/docs/ts/_cache/guide/pipes.jade

```diff
@@ -361,7 +369,7 @@ h3#async-pipe The impure #[i AsyncPipe]
   The component doesn't have to subscribe to the async data source,
   it doesn't extract the resolved values and expose them for binding,
   and the component doesn't have to unsubscribe when it is destroyed
-  (a potential source of memory leaks).
+  (a potent source of memory leaks).
```

Comment:
This change (potential -> potent) seems questionable, but since it's not yours, Patrice...

@kapunahelewong maybe you can fix it?